src/main/java/com/conveyal/gtfs/loader/JdbcTableWriter.java

```diff
@@ -789,9 +790,25 @@ private int updateStopTimesForPatternStops(List<PatternStop> patternStops) throw
         for (String tripId : timesForTripIds.keySet()) {
             // Initialize travel time with previous stop time value.
             int cumulativeTravelTime = timesForTripIds.get(tripId);
+            int timepointNumber = 0;
+            double timepointSpeed = 0;
+            double previousShapeDistTraveled = 0;
+            PatternStop currentTimepoint = patternStops.get(0); // First stop must be a timepoint
```

Comment:
I would add a check here to verify that the item exists and is a timepoint

Reply:
Good call, added in https://github.com/conveyal/gtfs-lib/pull/399/commits/b585c9df6e944b927859e7f6948dc5c2ab1080ff